src/Types.hs

```diff
@@ -476,16 +477,16 @@ data Phase =
 
 instance SOP.Generic Phase
 
-phaseName :: Phase -> ST
-phaseName = \case
-    PhaseWildIdea     -> "Wilde-Ideen-Phase"  -- FIXME: or is that unreachable code?
-    PhaseWildFrozen   -> "Wilde-Ideen-Phase"  -- FIXME: or is that unreachable code?
-    PhaseRefinement{} -> "Ausarbeitungsphase"
-    PhaseRefFrozen{}  -> "Ausarbeitungsphase"
-    PhaseJury         -> "Prüfungsphase"
-    PhaseVoting{}     -> "Abstimmungsphase"
-    PhaseVotFrozen{}  -> "Abstimmungsphase"
-    PhaseResult       -> "Ergebnisphase"
+instance HasUILabel Phase where
+    uilabel = \case
+        PhaseWildIdea     -> "Wilde-Ideen-Phase"  -- FIXME: or is that unreachable code?
+        PhaseWildFrozen   -> "Wilde-Ideen-Phase"  -- FIXME: or is that unreachable code?
```

Comment:
As I said, this is currently unreachable, but may be in the future and currently tests reach it.